test/libsolidity/semanticTests/revertStrings/empty_v1.sol

```diff
@@ -10,7 +10,7 @@ contract C {
 // ====
 // ABIEncoderV1Only: true
 // EVMVersion: >=byzantium
-// compileViaYul: false
+// compileViaYul: also
```

Comment:
This is intentionally disabled, because that implicitly turns this test to v2, while it is v1 only. There is another file with the `_v2` suffix for that. This applies to all `_v1` files.

Reply:
Sure, but if it works for both encoders?

Reply:
Well works now since that empty string encoding in v1 was fixed. The better option here is to just remove the v1 setting altogether, it is not needed. And perhaps just remove one of the files as there is no need for two.